modules/promotion/src/Plugin/Commerce/PromotionOffer/OrderItemPercentageOff.php

```diff
@@ -32,6 +32,7 @@ public function apply(EntityInterface $entity, PromotionInterface $promotion) {
       // @todo Change to label from UI when added in #2770731.
       'label' => t('Discount'),
       'amount' => $adjustment_amount->multiply('-1'),
+      'percentage' => $this->getAmount() * 100,
```

Comment:
Commerce always stores percentages in decimal form, so the multiplication by 100 is not needed